src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
so just handle it there, any 2xx is success, in particular with 204 do not parse the answer and get rid of all those specific handlings